frontend/src/utils/index.js

```diff
@@ -56,22 +17,6 @@ export default {
     const uniqueKey = `${item}-${index}`;
     return uniqueKey;
   },
-  getDateAsText(dateObject) {
```

Comment:
Are you sure these don't do anything? Last time I tried to remove them, I remember them being used to do something in the Meal Plan components.

Reply:
They used to, but I made sure I replaced their uses everywhere in favor of $d function. It's a standard function in vue-i18n, more powerful and natively translated.
https://kazupon.github.io/vue-i18n/guide/datetime.html

Reply:
Fancy! Thanks for all work on localization, it makes a huge difference.